tracer/util.go

```diff
@@ -75,3 +85,21 @@ func (rs *safeSource) Seed(seed int64) {
 	rs.source.Seed(seed)
 	rs.Unlock()
 }
+
+func UUIDToString(uuid uuid.UUID) string {
+	if val, err := uuid.MarshalBinary(); err != nil {
+		panic(err)
+	} else {
+		return hex.EncodeToString(val)
+	}
+}
```

Comment:
`uuid.MarshalBinary()` method never returns an error: https://github.com/google/uuid/blob/master/marshal.go

```suggestion
func UUIDToString(uuid uuid.UUID) string {
	bytes, _ := uuid.MarshalBinary()
	return hex.EncodeToString(bytes)
}
```

or even

```suggestion
func UUIDToString(uuid uuid.UUID) string {
	return hex.EncodeToString(uuid[:])
}
```

Reply:
I'd add a test for this function.

Reply:
done